src/locales/default.json

```diff
@@ -610,20 +600,50 @@
       "url": "https://blog.shutter.network/announcing-shutter-governance-shielded-voting-for-daos/"
     }
   },
+  "proposalValidation": {
+    "label": "Validation",
+    "title": "Select validation",
+    "settingsTitle": "Configure validation",
+    "paramPlaceholder": "Validation parameters",
+    "information": "The type of validation used to determine if a user can create a proposal. (Enforced on all future proposals)",
+    "executionError": "There was an error on our side and we could not verify if you are eligible to create a proposal. This is often due to a misconfigured voting validation or an unresponsive API involved in the validation.",
+    "notValidMessage": "Oops, you don't seem to be eligible to submit a proposal.",
+    "onlyMemberMessage": "You need to be a core member of the space in order to submit a proposal.",
+    "notConnectedMessage": "You need to connect your wallet in order to submit a proposal.",
+    "any": {
+      "label": "Anyone can create",
+      "description": "Anyone can create a proposal."
+    },
+    "basic": {
+      "label": "Basic",
+      "description": "Use any strategy to determine if a user can create a proposal.",
+      "invalidVoterMessage": "You do not meet the minimum balance requirement of {amount} {symbol} to create a proposal."
+    },
+    "passport-gated": {
+      "label": "Gitcoin Passport gated",
+      "description": "Protect your space from spam by requiring users to have a Gitcoin Passport to create a proposal.",
```

Comment:
Is there some way to make these optional. and if not configured, we read them from https://score.snapshot.org/api/validations

else we may have to ask everyone that create a validation to add a title and description here

Reply:
You can add/keep title and description for validations on strategies repo if you like, just on our UI I haven't found a good way to use them everywhere we need them. So for now we will have it like this